src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.LoggerFactoryWrapper.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+internal partial class RazorLanguageServer
+{
+    private class LoggerFactoryWrapper : ILoggerFactory, IRazorLoggerFactory
```

Comment:
if we had `IRazorLoggerFactory : ILoggerFactory` then this class definition would become simpler.

Reply:
This class doesn't actually need to implement ILoggerFactory at all, as it turns out